lib/plugins/ftp/index.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const log = require('intel').getLogger('sitespeedio.plugin.ftp');
+const PromiseFtp = require('promise-ftp');
+const merge = require('lodash.merge');
+const throwIfMissing = require('../../support/util').throwIfMissing;
+const fs = require('fs-extra');
+const defaultConfig = {
+  host: 'localhost',
+  port: 21,
+  user: 'user',
+  password: 'password'
+};
+function createFTPClient(ftpOptions) {
```

Comment:
Is this function used anywhere, i cannot find it? The ftpOptions is already defined in the function body as const, you can just rename it.